### PR TITLE
Add entry

### PR DESCRIPTION
**Pull request by ThysBat**
This is quite a long PR, I am sorry about that! But some files (stores with mock data) are almost identical.

### CODE REVIEW

File: stores/useArticleStore.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+//       addArticle: (newArticle) =>
+//         set({ articles: [...get().articles, newArticle] }),
+//       getArticlesByEntryId: (entryId) =>
+//         getArticlesByEntry(entryId, get().articles),
+//     }),
+//     {
+//       name: "articles",
+//     }
+//   )
+// );
```
Comment (dimitriosxmi) [inline stores/useArticleStore.js:78]: Prolly good to remove this no longer required code

File: stores/useEntryStore.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+//       createNewEntry,
+//       addEntry: (newEntry) => set({ entries: [...get().entries, newEntry] }),
+//       getEntriesByCategoryId: (categoryId) =>
+//         handleGetEntriesByCategoryId(categoryId, get().entries),
+//     }),
+//     {
+//       name: "entries",
+//     }
+//   )
+// );
```
Comment (dimitriosxmi) [inline stores/useEntryStore.js:82]: Prolly good to remove this no longer required code

File: stores/useArticleStore.js (hunk 1 above)
Comment (ThysBat) [reply stores/useArticleStore.js:78]: I thought about that, just wanted it to keep it a while if for some reason I want/ need to undo the immer middleware.

File: pages/project/[projectSlug]/category/[categorySlug]/entry/[entrySlug].js [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+  text-align: center;
+
+  border: none;
+  border-radius: var(--border-radius-s);
+
+  font-size: 2em;
+  margin: 1rem 0;
+  margin-left: 0;
+  margin-right: 0;
+  font-weight: bold;
```
Comment (dimitriosxmi) [inline pages/project/[projectSlug]/category/[categorySlug]/entry/[entrySlug].js:171]: the summary of all these  restylings looks to me like it should be this
```jsx
  margin: 1rem 0;
  padding: 0;

  width: 70%;
  opacity: 30%;

  font-size: 2em;
  font-weight: bold;
  text-align: center;

  border: none;
  border-radius: var(--border-radius-s);
```

**Review by dimitriosxmi — APPROVED** [review]
## LGTM ⭐ 
Notes: I've got only 1 complaint, and that is that you aren't using any comments, 😅 that makes the process of reviewing or further developing a bit more complicated 😋

File: pages/project/[projectSlug]/category/[categorySlug]/entry/[entrySlug].js (hunk 3 above)
Comment (ThysBat) [reply pages/project/[projectSlug]/category/[categorySlug]/entry/[entrySlug].js:171]: Oh damn, totally forgot about that little Oopsie 😅 Thanks for summarizing this mess! :D

**Comment by ThysBat** [comment]
Thanks for approving the code! You are totally right, I really have to improve on writing comments, especially when I use packages!

**Pull request merged**